### ISSUE TITLE
[BUG/ISSUE] KPP build_mechanism.sh looking for f90 instead of F90

### ISSUE THREAD

**Issue by rgryan92** [geoschem/geos-chem#713]
### Description of the problem
When trying to build a custom KPP mechanism in v13.0.0, build_mechamism.sh aborts.

The bug is in this section of build_mechanism.sh: 
```
if [[ ! -e gckpp_Rates.f90 ]]; then
  echo "KPP failure! Aborting."
  exit 1
fi
```
Latest downloaded version of gckpp_Rates.f90 is actually gckpp_Rates.F90 (capital F).
Build KPP works fine once the F is capitalised.

**Comment by yantosca** [comment]
Thanks for writing @rgryan92.  This should be fixed in 13.1.0 when the updated build_mechanism.sh is added.

In the future, we will add KPP as a submodule of GEOS-Chem, so that KPP updates can be pegged to specific GEOS-Chem versions.

**Comment by yantosca** [comment]
@rgryan: see https://kpp.readthedocs.io/en/latest/
